libusb/core.c

```diff
@@ -2204,6 +2207,111 @@ int API_EXPORTED libusb_set_auto_detach_kernel_driver(
 	return LIBUSB_SUCCESS;
 }
 
+/** \ingroup libusb_dev
+ * Check if the endpoint supports RAW_IO.
+ * \param dev_handle a device handle
+ * \param endpoint the endpoint to check
+ *
+ * \returns 1 if the endpoint supports RAW_IO
+ * \returns 0 if the endpoint does not support RAW_IO
+ * \returns a LIBUSB_ERROR code on failure
+ *
+ * Only endpoints using the WinUSB driver support RAW_IO,
+ * for all other backends/drivers this function will return 0.
+ *
+ * Fails if the interface the endpoint belongs to isn't claimed,
+ * \see libusb_claim_interface().
+ *
+ * \see libusb_endpoint_set_raw_io()
+ *
+ * Since version 1.0.28, \ref LIBUSB_API_VERSION >= 0x0100010B
```

Comment:
I guess this version number needs updating. In several places...

Reply:
Yes, shall we confidently put 1.0.30? I would say yes.

Reply:
I think it's mergeable.

Reply:
@seanm @HannesFranke-smartoptics : just regarding version number: what do you prefer:
a) I change it to 1.30 and merge
b) @HannesFranke-smartoptics change it (and rebase) then I merge
c) we merge as is and try not to forget to adapt the version number next time we do a release?

Reply:
> Personally I prefer (b), if he has not given up after over a year. Otherwise (a).

@seanm : OK, I'll do plan a) if @HannesFranke-smartoptics doesn't show up within a reasonable amount of time.

Reply:
@seanm @tormodvolden : please validate commit https://github.com/libusb/libusb/commit/2010b4f0f4b7f58847c73a6b340efbb19d31ef90 . 
This is what I plan to apply to this PR before merging. I prefer to have somebody validating the way I did it before merging it to master.

Reply:
Yes, absolutely, please amend this to 1.0.30 before merging. I am sure @HannesFranke-smartoptics has no objection to that fixup.